ui/StatusQ/src/StatusQ/Controls/StatusDropdown.qml

```diff
@@ -65,12 +65,12 @@ QC.Popup {
         True when:
           - \c bottomSheetAllowed is true, and
           - the window is in portrait (\c d.windowHeight > d.windowWidth), and
-          - the window width is at or below \c Theme.portraitBreakpoint.width.
+          - the window width is at or below \c ThemeUtils.portraitBreakpoint.width.
         Otherwise false. Used to switch layout/parenting to a full-width, bottom-anchored sheet.
     */
     readonly property bool bottomSheet: !bottomSheetAllowed ? false:
                                             d.windowHeight > d.windowWidth
-                                            && d.windowWidth <= Theme.portraitBreakpoint.width
+                                            && d.windowWidth <= ThemeUtils.portraitBreakpoint.width
```

Comment:
While implementing responsive components, I consistently felt that we’re missing a global property for this. Each component ends up checking the window width independently to determine whether it's in a compact/portrait mode, which leads to duplication and inconsistent logic.

Would it make sense to introduce a centralized property (something like _portraitMode_ or a general _compactMode_ flag) that all components can rely on? Having a single source of truth would make the responsive behavior more consistent across the entire app and simplify component logic.

Reply:
Yeah, I think it's good idea to consider using propagated property for that purpose. It will be easy to add to the Theme attached property and control from top-level main component, with possibility to adjust on any level if needed.